yml/apdb.yaml

```diff
@@ -1,10 +1,28 @@
 ---
 name: "ApdbSchema"
 "@id": "#apdbSchema"
+version: "0.1.1"
 description: The Alert Production Database (APDB) contains the catalogs resulting from
   image differencing during nightly Prompt Processing as well as the results of
   daily Solar System Processing.
 tables:
+- name: metadata
+  "@id": "#metadata"
+  description: Table containing various metadata key:value pairs for APDB.
+  columns:
+  - name: name
+    "@id": "#metadata.name"
```

Comment:
Should we call this `key`, in keeping with python dictionaries (which this effectively is), or is `name` the typical database term for this kind of key:value store?

Reply:
`key` is more natural, but in some database systems `key` is a reserved keyword and needs special care when used as column name (proper quoting). It's easier to avoid using keywords for column names.